### PR TITLE
ES High Availability example in OpenShift documentation is wrong

### PR DESCRIPTION
**Pull request by mburke5678**
https://bugzilla.redhat.com/show_bug.cgi?id=1702665

### CODE REVIEW

**Comment by vikram-redhat** [comment]
@anpingli PTAL

**Comment by anpingli** [comment]
Why we mix up the quorum and replicas together? 

if replicas=1, the replica shard will be created and locate in the other nodes.that needs at least two nodes in the cluster.
if replicas=3, two replica shards will be created and locate in different nodes, that need at least three nodes in the cluster.

To prevent split-brain, it is better to use an odd number, to provide toleration and auto-recovery ability,  n/2+1 nodes should be alive in the ES cluster.  so when replicas=1, the cluster number should >=3. when replics=2, the cluster node number should >=5.

**Comment by mburke5678** [comment]
@anpingli I believe this section is intended to describe how using multiple replicas can help if some of your nodes go down. 

If you increase the number of replicas, you are less likely to lose data if nodes go down. However, there is a trade-off as more replicas require more resources.

**Review by anpingli — COMMENTED** [review]
/lgtm

**Comment by mburke5678** [comment]
Preview: http://file.rdu.redhat.com/~mburke/BZ-1702665/install_config/aggregate_logging.html#aggregated-elasticsearch

Aggregating Container Logs -> Elasticsearch -> Configuring Elasticsearch for High Availability

File: install_config/aggregate_logging.adoc [hunk 1]
```diff
@@ -814,13 +814,13 @@ to a value of `1` or higher to create replicas.
 
 Use the following scenarios as a guide for an {product-title} cluster with three Elasticsearch nodes:
 
-* If you can tolerate one Elasticsearch node going down,
-set `openshift_logging_es_number_of_replicas` to `1`. This ensures
-that two nodes have a copy of all of the Elasticsearch data in the cluster.
+* With `openshift_logging_es_number_of_replicas` set to `1`, two nodes have a copy of all of the Elasticsearch data in the cluster.
+This ensures that if a node with the Elasticsearch data goes down, another node has a copy of all of the Elasticsearch data in the cluster.
```
Comment (bergerhoffer) [inline install_config/aggregate_logging.adoc:818]: Probably fine, but just wondering if "if a node with the Elasticsearch data" should be "if a node with Elasticsearch data"

File: install_config/aggregate_logging.adoc [hunk 2]
```diff
@@ -814,13 +814,13 @@ to a value of `1` or higher to create replicas.
...
-set `openshift_logging_es_number_of_replicas` to `1`. This ensures
-that two nodes have a copy of all of the Elasticsearch data in the cluster.
+* With `openshift_logging_es_number_of_replicas` set to `1`, two nodes have a copy of all of the Elasticsearch data in the cluster.
+This ensures that if a node with the Elasticsearch data goes down, another node has a copy of all of the Elasticsearch data in the cluster.  
 
-* If you must tolerate two Elasticsearch nodes going down,
-set `openshift_logging_es_number_of_replicas` to `2`. This ensures that
-every node has a copy of all of the Elasticsearch data in the cluster.
+* With `openshift_logging_es_number_of_replicas` set to `3`, four nodes have a copy of all of the Elasticsearch data in the cluster.
+This ensures that if three nodes with the Elasticsearch data go down, one node has a copy of all of the Elasticsearch data in the cluster.
```
Comment (bergerhoffer) [inline install_config/aggregate_logging.adoc:821]: Same question on if the "the" is necessary here.

**Pull request merged**